Gems/Atom/Feature/Common/Code/Source/DiffuseGlobalIllumination/DiffuseProbeGrid.cpp

```diff
@@ -245,6 +245,23 @@ namespace AZ
                 m_bakedClassificationImage.get();
         }
 
+        void DiffuseProbeGrid::ResetCullingVisibility()
+        {
+            m_cullable.m_isVisible = false;
+        }
+
+        bool DiffuseProbeGrid::GetIsVisible() const
```

Comment:
Minor, but I would personally think that IsVisible() reads better than GetIsVisible()